vulcan/lib/server/workflows/scripts/retrieve_selection_options.py

```diff
@@ -1,42 +1,35 @@
-from archimedes.functions.dataflow import input_var, output_json
+from archimedes.functions.dataflow import input_var, input_json, output_json, parseModelAttr, buildTargetPath
 from archimedes.functions.magma import question, connect
-from archimedes.functions.list import unique
+from archimedes.functions.list import unique, flatten
+from archimedes.functions.environment import project_name
 
-seq_model_name = "sc_seq"
-seq_pool_model_name = "sc_seq_pool"
+pdat = input_json("project_data")[project_name]
+selection_options = list(pdat['selection_options'].values())
+
+seq_target = parseModelAttr(pdat['seq_h5_counts_data'])
+q_start = [
+    seq_target['model'],
+    [ '::has', seq_target['attribute']],
+    '::all'
+]
 
 magma = connect()
 
-experiments = question(
+select1 = question(
```

Comment:
Do these need to check if `select#` is defined for the given project, if it has fewer than 3?